src/components/shared/Input/Input.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Input.scss';
+
+const Input = ({ setValueToState, label, ...otherProps }) => {
```

Comment:
a jak dziala ten setValueToState?

Reply:
to jest do poprawy, we wczesniejszej wersji inputa z tego korzystalem i nie usunołem podczas refactoringu.   dobrze ,  że to wychaczyłeś. dzięki!